tests/rules/unqualified-attributes.js

```diff
@@ -13,6 +13,16 @@
             Assert.areEqual("Unqualified attribute selectors are known to be slow.", result.messages[0].message);
         },
 
+        "A selector that contains a class is considered qualified": function(){
+            var result = CSSLint.verify("p [href].fancy[type=text] { font-size: 10px; }", {"unqualified-attributes": 1 });
```

Comment:
Maybe make this simpler so it is just testing the class bit and not the rest, e.g.:

```
.fancy[type=text] { font-size: 10px; }
```